### DISCUSSION TITLE
[Dev Tools] Explorer not indexing events on verified contract

### DISCUSSION THREAD

**Discussion by githubdoramon** [zkSync-Community-Hub/zksync-developers#844] [Dev Tooling] [answered]
### Team or Project

Sophon

### Environment

Mainnet

### Select the Dev Tool you are using

Block Explorers

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

The following tx is being sent to a verified contract, but the logs are not parsed

https://explorer.sophon.xyz/tx/0x1c0e4aef286914a6a2ef652854107fde248a09724e4feb06583c1d8b93dfc152

Maybe because it is a proxy?

**Comment by alhenaai** [comment]
It appears that you are encountering issues with transaction logs not being parsed on the Sophon block explorer. Unfortunately, I don't have specific information on troubleshooting issues related to the Sophon block explorer or its transaction log parsing capabilities.

For more detailed support, I recommend visiting the ZKsync developers discussions on Github, where you can engage with other developers and possibly find solutions or workarounds for your issue. Here's the link to the discussions: [ZKsync Developers Discussions](https://github.com/zkSync-Community-Hub/zksync-developers/discussions).

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by cytadela8** [comment, marked as answer]
Hi, I have confirmed with the Explorer team that this is unfortunately a known issue where we do not parse logs for proxy contracts. Fixing this is planned.

**Reply by JackHamer09** [reply]
@cytadela8 any chance you could also create an issue on block explorer repo to track the issue?

**Reply by cytadela8** [reply]
[This is the issue](https://github.com/matter-labs/block-explorer/issues/361) in block-explorer repo
